lib/page/needed.rb

```diff
@@ -12,15 +12,15 @@ def title
     end
 
     def to_find
-      issues 'New Country,To Find'
+      (issues 'New Country,To Find').sort_by(&:title)
```

Comment:
I think it would be much cleaner to add this to the `issues` method, rather than every method that calls that.